core/src/test/java/org/apache/iceberg/avro/AvroTestHelpers.java

```diff
@@ -95,7 +94,7 @@ static void assertEquals(Types.ListType list, List<?> expected, List<?> actual)
   static void assertEquals(Types.MapType map, Map<?, ?> expected, Map<?, ?> actual) {
     Type valueType = map.valueType();
 
-    Assert.assertEquals("Map size should match", expected.size(), actual.size());
+    Assertions.assertThat(actual).as("Map keys should match").hasSize(expected.size());
```

Comment:
```suggestion
    Assertions.assertThat(actual).as("Map keys should match").hasSameSizeAs(expected);
```
same for the above one

Reply:
I've modified the code based on your suggestions.